src/containers/SearchPage/components/SearchResultItem.tsx

```diff
@@ -25,11 +27,25 @@ import {
 import { SafeLink } from "@ndla/safelink";
 import { styled } from "@ndla/styled-system/jsx";
 import { linkOverlay } from "@ndla/styled-system/patterns";
-import { ContentTypeBadgeNew } from "@ndla/ui";
+import { ContentType } from "@ndla/ui";
 import { ContentTypeFallbackIcon } from "../../../components/ContentTypeFallbackIcon";
 import { DialogCloseButton } from "../../../components/DialogCloseButton";
 import { SearchItem } from "../searchHelpers";
 
+const contentTypeToVariantMapping = {
```

Comment:
halvveis kopi fra packages

Reply:
Hadde vært nice om vi kunne hatt et sted denne typen logikk hadde vært eksportert i frontend-packages, sånn at det kunne blitt brukt både for denne komponenten og for badge 🤔 vil jo tro vi ønsker å bruke de samme fargene for de ulike ressurstypene alle steder?

Reply:
Jeg måtte utvide dette for akkurat denne delen men ja, jeg er helt enig med deg. kan ta det i egen PR